tests/nightly/test_large_vector.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Will this test pass with your Part-1 PR? If not, please check it in together with your Part-2 without causing nightly test failure.